tests/algorithms/test_required_on_load.py

```diff
@@ -85,15 +85,17 @@ def test_idempotent(algo_name: str):
         if algo_name == 'StochasticDepth':
             original_model = composer_resnet(model_name='resnet50')
         elif algo_name in ['Alibi', 'GatedLinearUnits']:
-            pytest.importorskip('transformers')
-            original_model = create_bert_classification()
+            transformers = pytest.importorskip('transformers')
+            from composer.models import HuggingFaceModel
+            hf_model = transformers.AutoModelForSequenceClassification.from_config(tiny_bert_config)
+            original_model = HuggingFaceModel(hf_model, use_logits=True)
```

Comment:
why this change?

Reply:
create_bert_classification (with no args) calls out to huggingface hub. This avoids that, same as the other HF tests